lib/plugin/instance.rb

```diff
@@ -137,11 +136,15 @@ def full_admin_route
   end
 
   def any_settings?
```

Comment:
Is this method still used now? 🤔

Reply:
Yah still using the serializer 

https://github.com/discourse/discourse/blob/dcf0cdf6975464a3539093eb3d8db391ae2ee93a/app/serializers/admin_plugin_serializer.rb#L67-L69